pennylane/operation.py

```diff
@@ -420,8 +420,9 @@ class Operator(abc.ABC):
         params (tuple[tensor_like]): trainable parameters
         wires (Iterable[Any] or Any): Wire label(s) that the operator acts on.
             If not given, args[-1] is interpreted as wires.
-        do_queue (bool): indicates whether the operator should be
-            recorded when created in a tape context
+        do_queue (bool): indicates whether the operator should be recorded when created in
+            a tape context. This arguement is deprecated, instead of setting it to ``False``
+            use :meth:`~.QueuingManager.stop_recording`.
```

Comment:
I really like the explicit statement of which action before the deprecation should be replaced by which action after the deprecation! Maybe we could be as explicit in the changelog and the deprecations doc page? :)

Reply:
Good point. Done. 94938879d06f40ec63a5a2078308be04e250fe55